config/migrations/add-concept-schemes.sparql.disabled

```diff
@@ -0,0 +1,66 @@
+INSERT DATA {
```

Comment:
This file should be checked as the content in here was added to the database in the `onLoad` folder. These items do not have a `concept-scheme` (parent) or do not have `concepts` (childs) under the `concept-scheme`  (parent).